src/index.js

```diff
@@ -101,19 +102,18 @@ export const getTargets = (targets = {}) => {
     targetOps.node = getCurrentNodeVersion();
   }
 
-  // Rewrite Electron versions to their Chrome equivalents
+  // Replace Electron versions with their Chrome equivalent
   if (targetOps.electron) {
     const electronChromeVersion = parseInt(electronToChromium(targetOps.electron), 10);
 
-    if (!electronChromeVersion) {
-      throw new Error(`Electron version ${targetOps.electron} is either too old or too new`);
-    }
+    invariant(
+      !!electronChromeVersion,
+      `Electron version ${targetOps.electron} is either too old or too new`
+    );
```

Comment:
Maybe let's place this stuff to `normalize-options` module like other options' validations?